spec/controllers/cuisines_controller_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe CuisinesController, type: :controller do
```

Comment:
Minor - I would remove this empty test files and move them into a dedicated `tests` branch which you'll open a PR for later.
They don't add anything to the codebase right now so they shouldn't be in this PR